schemas/adaptive-card.json

```diff
@@ -956,6 +1224,62 @@
 				"title"
 			]
 		},
+		"FontType": {
+			"type": "string",
+			"version": "1.2",
+			"description": "Type of font to use for rendering",
```

Comment:
>Type of font to use for rendering [](start = 19, length = 33)

I think "rendering" is an odd word here. I think simply "The type of font to use" is sufficient? Technically everything is "for rendering" :)